src/utilstrencodings.cpp

```diff
@@ -19,7 +19,7 @@ static const string CHARS_ALPHA_NUM = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNO
 
 static const string SAFE_CHARS[] =
 {
-    CHARS_ALPHA_NUM + " .,;_/:?@()", // SAFE_CHARS_DEFAULT
+    CHARS_ALPHA_NUM + " .,;-_/:?@()", // SAFE_CHARS_DEFAULT
```

Comment:
Hmm, does this addition make sense for all the other uses of `SAFE_CHARS_DEFAULT`? Or should we add a new rule specifically for version fields?

Reply:
I think you are right, added a new rule and force pushed.